src/test/kotlin/ComponentTest.kt

```diff
@@ -0,0 +1,86 @@
+import constant.StringConstant
+import io.kotest.core.spec.style.ExpectSpec
+import io.kotest.matchers.shouldBe
+import utilities.StringBasedCalculator
+import kotlin.math.exp
+
+class ComponentTest : ExpectSpec({
+
+    context("component test: StringBasedCalculator") {
+        expect("수식 분리 테스트") {
+
+            val calculator = StringBasedCalculator("1 + 3 - 2 * 4 / 8")
+
+            println(calculator.numbers)
+            println(calculator.operators)
+
+            (calculator.numbers.size ?: 0) shouldBe 5
+            (calculator.operators.size ?: 0) shouldBe 4
```

Comment:
단언문에서 Elvis연산자를 쓰는 것 자체가 로직의 결과를 유추하지 못한다고 보여질 수 있어요. 
```suggestion
            calculator.numbers shouldNotBe null
            calculator.numbers shouldHaveSize 5
```

Reply:
제거하도록 하겠습니다